### PR TITLE
Fix scaleio tests

### PR DESCRIPTION
**Pull request by vishesh92**
### Description

This PR fixes code coverage of scaleio.

The other plugins have the same setting.

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [ ] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [ ] Minor
- [ ] Trivial

### CODE REVIEW

File: plugins/storage/volume/scaleio/src/test/java/org/apache/cloudstack/storage/datastore/client/ScaleIOGatewayClientImplTest.java [hunk 1]
```diff
@@ -48,7 +48,7 @@
 
 @RunWith(MockitoJUnitRunner.class)
 public class ScaleIOGatewayClientImplTest {
-    private final int port = 443;
+    private final int port = 8443;
```
Comment (vishesh92) [inline plugins/storage/volume/scaleio/src/test/java/org/apache/cloudstack/storage/datastore/client/ScaleIOGatewayClientImplTest.java:51]: `443` is a restricted port in linux and needs root access. Used `8443` instead.

**Comment by vishesh92** [comment]
image

File: plugins/storage/volume/scaleio/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker [hunk 2]
```diff
@@ -0,0 +1 @@
+mock-maker-inline
```
[the remark this answers is not in the document]
Comment (DaanHoogland) [reply plugins/storage/volume/scaleio/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker:1]: also the file should be added to the rat exclusion list. Not sure if a wildcard for paths is possible

File: pom.xml [hunk 3]
```diff
@@ -1063,6 +1063,7 @@
                             <exclude>ui/legacy/**</exclude>
                             <exclude>utils/testsmallfileinactive</exclude>
                             <exclude>utils/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker</exclude>
+                            <exclude>plugins/storage/volume/scaleio/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker</exclude>
```
Comment (DaanHoogland) [inline pom.xml:1066]: would this work?
```suggestion
                            <exclude>**/mockito-extensions/org.mockito.plugins.MockMaker</exclude>
```

Comment (vishesh92) [reply pom.xml:1066]: Let me test this.

Comment (vishesh92) [reply pom.xml]: This works.

**Review by rohityadavcloud — APPROVED** [review]

**Comment by rohityadavcloud** [comment]
@blueorangutan package

**Review by DaanHoogland — APPROVED** [review]
clgtm

**Comment by DaanHoogland** [comment]
only tests and pom.xmls changed, as this is building I'm m erging

**Pull request merged**